discovery-provider/alembic/versions/281de8af4b93_add_indexes.py

```diff
@@ -0,0 +1,41 @@
+"""add indexes
+
+Revision ID: 281de8af4b93
+Revises: ffcb2df7b0ee
+Create Date: 2020-09-28 15:27:40.389787
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '281de8af4b93'
+down_revision = 'ffcb2df7b0ee'
+branch_labels = None
+depends_on = None
+
+def upgrade():
+    # Add an index for track owner id on the tracks table
+    op.create_index(op.f('track_owner_id_idx'),
+                'tracks', ['owner_id'], unique=False)
+
+    # Update the index on the aggregate_plays materialized view to be unique for concurrent updates
+    connection = op.get_bind()
+    connection.execute('''
```

Comment:
have you run this on a db dump?

Reply:
Yeah, I ran this on a prod snapshot I booted up.